### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by hwrdtm**
# What

Not grabbing the transactionHash from the TransactionReceipt correctly.

### CODE REVIEW

File: routes/auth/claim.ts [hunk 1]
```diff
@@ -33,10 +33,10 @@ export async function mintClaimedKeyId(
 			authMethodPubkey: "0x",
 		});
 		console.info("claimed key id: transaction hash (request id): ", {
-			requestId: mintTx.hash
+			requestId: mintTx.transactionHash,
```
Comment (Ansonhkg) [inline routes/auth/claim.ts:36]: if `.hash` was working before shouldn't we try to return either/or result?

**Review by Ansonhkg — APPROVED** [review]

File: routes/auth/claim.ts (hunk 1 above)
Comment (joshLong145) [reply routes/auth/claim.ts:36]: why do we need this change? arent both hash values identical for the tx hash?

Comment (hwrdtm) [reply routes/auth/claim.ts:36]: @joshLong145 .hash was actually undefined when I was using it. Was it the same for you?

**Review by joshLong145 — APPROVED** [review]

File: routes/auth/claim.ts (hunk 1 above)
Comment (joshLong145) [reply routes/auth/claim.ts:36]: yes after testing again i see `hash` was undefined. merging this in as the type change is required after calling `tx.wait()` on the returned transaction.

**Pull request merged**